bundles/CoreBundle/Migrations/Version20230616085142.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Db;
+
+final class Version20230616085142 extends AbstractMigration
+{
+    private const ID_COLUMN = 'o_id';
+    private const PK_COLUMNS = '`' . self::ID_COLUMN .
```

Comment:
i'd keep `PK_COLUMNS` as an array and use a 
```
UNIQUE (`' . implode('`,', SELF:PK_COLUMNS) . '`)'
```
but not that important change, just looks easier to read and "mantain" the array (and sonarcloud line char length, breaking in multiple lines)

Reply:
I also thought about this, but then it's harder to read in the sql string

Reply:
ok, fine with me, looks something that is not going to be changed so often anyways